### ISSUE TITLE
The colour of cloth is not displayed correctly in section "More information".

### ISSUE THREAD

**Issue by kindaro** [DFHack/dfhack#1391]
I observe it with pig tail cloth and dimple dye. I do not know if other types of items are affected. 

#### Try it yourself:

1. Obtain some pig tail cloth.
2. Dye it blue with dimple dye.
3. Take a look at it.

#### Expected:

1. The cloth is displayed with a blue icon on the map.
2. If I press `k - ⏎`, it reads `Color: blue` in the section `More information (DFHack):`.
3. If I press `v` for "Description", I can see again that it is midnight blue.

#### Actual:

1. As expected.
2. It reads "Color: grey".
3. As expected.

* * *

I use Dwarf Fortress 0.44.12 and DFHack built from `develop`, commit `e56cb2a2`.

**Comment by kindaro** [comment]
@BenLubar Must be the case. I would much prefer it showed the actual colour though. After all, one cannot undye things, so the information concerning their previous colour is hardly of value.

**Comment by BenLubar** [comment]
Is dye the only type of improvement that changes the color of an item?

**Comment by kindaro** [comment]
@BenLubar Hard to know. Even if currently so, Toady One may add another colour changing feature in the next release. I think we should aim at best effort, and make the script recognize the dyes for now.

**Comment by BenLubar** [comment]
Ideally, we'd use this vmethod, but I don't think we can create a `std::vector<int16_t>` from Lua:

https://github.com/DFHack/df-structures/blob/fb68dc5a49ce40bebe52ce324699c73c888963b7/df.items.xml#L608-L611

**Comment by lethosor** [comment]
Yeah, it doesn't appear to be possible. Maybe a wrapper in the Items module would be good to have.
